sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/DescribeTableSuite.scala

```diff
@@ -436,12 +436,14 @@ trait DescribeTableSuiteBase extends command.DescribeTableSuiteBase
            |  id INT
            |)
            |USING parquet COMMENT 'table_comment'
+           |DEFAULT COLLATION UTF8_BINARY
```

Comment:
QQ: is there such a test case in V2?

Reply:
currently OSS `Desc As Json` support is only implemented for v1, but we can move some tests to `DescribeTableSuiteBase` for future test coverage